help_to_heat/frontdoor/live_settings.py

```diff
@@ -0,0 +1,30 @@
+from help_to_heat.frontdoor.models import LiveSettings
+
+
+def public_portal_is_closed():
+    return not public_portal_is_open()
+
+
+def public_portal_is_open():
+    live_settings = _get_or_create_live_settings()
+    return live_settings.public_portal_open
+
+
+def set_public_portal_open(open):
+    live_settings = _get_or_create_live_settings()
+    live_settings.public_portal_open = open
+    _save_live_settings(live_settings)
+
+
+def _get_or_create_live_settings():
+    if not LiveSettings.objects.exists():
+        live_settings = LiveSettings()
+        _save_live_settings(live_settings)
```

Comment:
this is potentially dangerous if two users try to generate live settings at once, could lead to two rows in this table. have added to ticket release notes to check the database manually & ensure this is not the case

once it exists in the db we never generate a new LiveSettings object so should be safe going into the future